paasta_tools/marathon_tools.py

```diff
@@ -135,6 +135,9 @@ def __init__(self, current: List[MarathonClient], previous: List[MarathonClient]
         self.current = current
         self.previous = previous
 
+    def __repr__(self) -> str:
+        return 'Current: %s | Previous: %s' % (self.current, self.previous)
```

Comment:
I think usually it's good to have __repr__ return something that would more or less evaluate to the real thing, so I'd prefer `'MarathonClients(current=%r, previous=%r)' % (self.current, self.previous)`